### PR TITLE
std prk cfg fls

### PR DESCRIPTION
**Pull request by goern**
## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

just added some standard project config files.

### CODE REVIEW

**Comment by goern** [comment]
@pacospace please have a look at the checks here, it doesnt seem to finish

File: Pipfile [hunk 1]
```diff
@@ -12,3 +16,6 @@ faust = "*"
 
 [requires]
 python_version = "3.6"
+
+[pipenv]
+allow_prereleases = true
```
Comment (fridex) [inline Pipfile]: Do we want this?

File: .thoth.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-host: test.thoth-station.ninja
+host: stage.thoth-station.ninja
```
Comment (fridex) [inline .thoth.yaml]: `khemenu`

File: Pipfile (hunk 1 above)
Comment (goern) [reply Pipfile]: hmm, no, but dont we need it for black?!

Comment (fridex) [reply Pipfile]: Do we want to have black as part of the application stack?

**Pull request merged**